SWLOR.Game.Server/Feature/ShipModuleDefinition/MissileLauncherModuleDefinition.cs

```diff
@@ -97,10 +103,14 @@ private void MissileLauncher(
                 {
                     var moduleDamage = dmg + moduleBonus;
                     // Missiles do 25% more damage to unshielded targets. Due to shield recharge starting instantly, allow for up to 4 shield points to be considered "unshielded".
-                    if (targetShipStatus.Shield <= 4)
+                    if (targetShipStatus.Shield <= 8)
                     {
                         moduleDamage += moduleDamage / 4;
                     }
+                    if (targetShipStatus.Shield >= 9)
+                    {
+                        moduleDamage -= moduleDamage / 5;
```

Comment:
You're subtracting damage here. Expected?

Reply:
Yes, they do less damage to shields.
Ions Normal damage to shields, reduced to hull. (But higher damage on shields overall by base)
Missiles do Less to shields more to hull. 
Thermals just chill in the middle damage wise.